algorithms/appo/model_utils.py

```diff
@@ -274,6 +282,8 @@ def create_standard_encoder(cfg, obs_space, timing):
         encoder = ResnetEncoder(cfg, obs_space, timing)
     elif cfg.encoder_type == 'mlp':
         encoder = MlpEncoder(cfg, obs_space, timing)
+    elif cfg.encoder_type == 'mujoco':
```

Comment:
Again, why do we need the MujocoEncoder? MlpEncoder will do!

Reply:
My mistake, I forgot to delete this.